index.html

```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>SEE PROJECT</button>
+                      </div>
+                </div>
+            </div>
+
+            <div class="grid-item">
+                <div class="flex-container">
+                      <div class="works-image"></div>
+                      <div class="works-inner">
+                          <p>Multi-Post Stories<br>Gain+Glory</p>
+                          <ul>
+                              <li><a href="#">Ruby on rails</a></li>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>SEE PROJECT</button>
+                      </div>
+                </div>
+            </div>
+
+            <div class="grid-item">
+                <div class="flex-container">
+                      <div class="works-image"></div>
+                      <div class="works-inner">
+                          <p>Multi-Post Stories<br>Gain+Glory</p>
+                          <ul>
+                              <li><a href="#">Ruby on rails</a></li>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>SEE PROJECT</button>
+                      </div>
+                </div>
+            </div>
+
+            <div class="grid-item">
+                <div class="flex-container">
+                      <div class="works-image"></div>
+                      <div class="works-inner">
+                          <p>Multi-Post Stories<br>Gain+Glory</p>
+                          <ul>
+                              <li><a href="#">Ruby on rails</a></li>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>SEE PROJECT</button>
+                      </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about-myself">
+            <div>
+                <h2>About me</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <span><a href="#">Get My Resume</a></span>
+            </div>
```

Comment:
Your about section looks a little bit different from the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=WZMqc3ufIF1I5Dt6-0). 

your app
<img width="278" alt="yourAbout" src="https://user-images.githubusercontent.com/64643995/212410689-94e7f795-14a6-4145-8474-21da607843b7.png">

the design
<img width="289" alt="designAbt" src="https://user-images.githubusercontent.com/64643995/212410745-42f2cde1-4ebb-43e0-83f8-e080521ab217.png">

kindly make the necessary changes. consider the background image, font properties, and spacing.🤗